packages/website/src/remark/tabs.license.md

```diff
@@ -0,0 +1,23 @@
+original code https://github.com/mccleanp/remark-docusaurus-tabs
```

Comment:
Why not use the original code directly? Should we file an issue to look at making a fork / helping take ownership / etc?

Reply:
i should actually make pr in there

Reply:
i did push pr to this package, https://github.com/mccleanp/remark-docusaurus-tabs/pull/2